bot/commands/misc/convert.js

```diff
@@ -78,8 +79,8 @@ module.exports = new Command('convert', async (msg, args, context) => {
 		try {
 			// if it can't convert it normally, try to find it as a currency, throws if it can't find a currency either
 			message = await convertCurrency(baseValue, baseType, targetType);
-		} catch (__) {
-			message = `Couldn't convert from ${baseValue} ${baseType} to ${baseValue}. Type \`${context.prefix}convert\` to see all available units.`;
+		} catch (error) {
```

Comment:
Keep the caught error as `__` if we're not using it - clearer that the value isn't being used, and in the future I'm planning to introduce an ESLint rule that will complain about unused variables that aren't marked with underscores like this.
```suggestion
		} catch (__) {
```

Alternatively if you wanna add a log for the error here I'm not opposed - depends on whether or not you see it being helpful in the future.